app/jobs/import_legislators_job.rb

```diff
@@ -0,0 +1,57 @@
+class ImportLegislatorsJob
+  include Worker
+
+  def initialize(redis = Redis.new, service = OpenStatesService.new)
+    @redis = redis
+    @service = service
+  end
+
+  def perform
+    import_date = @redis.get(:import_legislators_job_date)&.to_time
+
+    legislator_attrs = @service
+      .fetch_legislators(fields: fields, updated_since: import_date)
+      .map { |data| parse_attributes(data) }
+      .reject(&:nil?)
+
+    legislator_attrs.each do |attrs|
+      legislator = Legislator.upsert_by!(:external_id, attrs)
+    end
+
+    @redis.set(:import_legislators_job_date, Time.zone.now)
+  end
+
+  private
+
+  def parse_attributes(data)
+    attrs = {
+      external_id: data['id'],
```

Comment:
`external_id` is the the id from the ilga.gov sites, and at some point should probably be called `ilga_id`. i don't think we can set it through this service.

Reply:
Any idea what that looks like? Is it something we need to scrape then?